clients/privacy-center/cypress/fixtures/consent/experience_none.json

```diff
@@ -0,0 +1,6 @@
+{
+  "items": [],
```

Comment:
Added this fixture to ensure we had a mock response for an empty experience, not matching any location.